### PR TITLE
OPENEUROPA-1185: Content broker configuration module

### PR DESCRIPTION
**Pull request by ademarco**

### CODE REVIEW

**Comment by voidtek** [comment]
The step
`Then I should see the "<title>" in the title region` is a duplicate of `And I should see the heading "<title>" in the "page header" region`

We can remove the following function because it's not generic and we have a region hardcoded.
```
public function assertTitleInCorrectRegion(string $title): void {
    $this->assertSession()->elementTextContains('css', '.ecl-page-header h1', $title);
  }
```

File: sites/energy/runner.yml.dist [hunk 1]
```diff
@@ -32,7 +32,18 @@ drupal:
           driver: 'sparql'
     config:
       oe_content.settings:
-          provenance_uri: 'http://ec.europa.eu/energy'
+        provenance_uri: 'http://ec.europa.eu/energy'
+      europa_demo_content_broker.settings:
+        topics:
+          -
+            site: 'http://ec.europa.eu/inea'
+            topics: []
```
Comment (voidtek) [inline sites/energy/runner.yml.dist]: According to the document "scenario V2":
- "Energy" should be configured to be subscribed to all Announcements from sites "Inea" and "Info" 
- "Energy" should be configured to be subscribed to all Events from "Inea" only

**Comment by voidtek** [comment]
> The step
> `Then I should see the "<title>" in the title region` is a duplicate of `And I should see the heading "<title>" in the "page header" region`
> 
> We can remove the following function because it's not generic and we have a region hardcoded.
> 
> ```
> public function assertTitleInCorrectRegion(string $title): void {
>     $this->assertSession()->elementTextContains('css', '.ecl-page-header h1', $title);
>   }
> ```

After talking with @upchuk, I understand why we need to create another function.
Should we put out the name of the class ( in the behat.yml file) ?

File: sites/energy/runner.yml.dist (hunk 1 above)
Comment (voidtek) [reply sites/energy/runner.yml.dist]: According to https://github.com/ec-europa/europa-demo/pull/34/files#diff-b490d2035999cc5da86e05b43a00e6f1R37 the configuration is fine.

**Comment by ademarco** [comment]
@voidtek we could use the transformation context for pages but its regular expression is too poor, we would need to improve that. I've opened a follow up here and left a comment in this PR's code: https://webgate.ec.europa.eu/CITnet/jira/browse/OPENEUROPA-1224

**Pull request merged**